### ISSUE TITLE
Add Alarms

### ISSUE THREAD

**Issue by pcwii** [don86nl/ha_intents#1]
This is not so much as a bug as a request. 
Any chance alarms can be created?
"Set an alarm for 10 PM"
Then play an mp3 or a audible announcement when the alarm is reached.

**Comment by don86nl** [comment]
Yes I think that is possible. It is already part of the nighttime routine. I'll add it soon.

**Comment by don86nl** [comment]
Have you tried the package '[assist_alarmtime.yaml](https://github.com/don86nl/ha_intents/blob/main/config/packages/assist_alarmtime.yaml)' ?
